v2.1/create-security-certificates-custom-ca.md

```diff
@@ -0,0 +1,117 @@
+
+For secure clusters, you can avoid getting the warning message by using a certificate issued by a public CA:
+
+1. Request a certificate from a public CA (for example, [Let's Encrypt](https://letsencrypt.org/)). The certificate must have the IP addresses and DNS names used to reach the Admin UI listed in the `Subject Alternative Names` field.
+2. Rename the certificate and key as `ui.crt` and `ui.key`.
+3. Add the `ui.crt` and `ui.key` to the [certificate directory](create-security-certificates.html#certificate-directory). `ui.key` must not have group or world permissions (maximum permissions are 0700, or rwx------). This check can be disabled by setting the environment variable `COCKROACH_SKIP_KEY_PERMISSION_CHECK=true`.
+4. For nodes that are already running, load the `ui.crt` certificate without restarting the node by issuing a `SIGHUP` signal to the cockroach process:
+   {% include copy-clipboard.html %}
+   ~~~ shell
+   pkill -SIGHUP -x cockroach
+   ~~~
+   The `SIGHUP` signal must be sent by the same user running the process (e.g., run with sudo if the cockroach process is running under user root).
+
+### Node key and certificates
+
+File name pattern | File usage
+-------------|------------
+`ca.crt`     | CA certificate
+`node.crt`   | Server certificate. <br><br> `node.crt` must have `CN=node` and the list of IP addresses and DNS names listed in `Subject Alternative Names` field. <br><br>Must be signed by `ca.crt`
+`node.key`   | Key for server certificate
+`ui.crt` | UI certificate. `ui.crt` must have the IP addresses and DNS names used to reach the Admin UI listed in `Subject Alternative Names`
+`ui.key` | Key for the UI certificate
+
+### Client key and certificates
+
+File name pattern | File usage
+-------------|------------
+`ca.crt`     | CA certificate
+`client.<user>.crt` | Client certificate for `<user>` (e.g., `client.root.crt` for user `root`) and must be signed by `ca.crt`. Each `client.<user>.crt` must have `CN=<user>`  (for example, `CN=marc` for `client.marc.crt`)
+`client.<user>.key` | Key for the client certificate
+
+## Split node certificates
+
+The node certificate discussed in the `cockroach cert` command documentation is multifunctional, which means the same certificate is presented when the node acts as a server as well as a client. To make the certificate multi-functional, the `node.crt` created using the `cockroach cert` command has `CN=node` and the list of IP addresses and DNS names listed in `Subject Alternative Names` field. This works if you are also using the CockroachDB CA created using the `cockroach cert` command. However, if you need to use an external public CA or your own organizational CA, the CA policy might not allow it to sign a node certificate containing a CN that is not an IP address or domain name.
```

Comment:
I think that last sentence should say "CA policy might not allow it to sign a *server* certificate containing a CN that is not an IP address". In this configuration we still require the node *client* certificate to have CN=node.